Doc/whatsnew/3.10.rst

```diff
@@ -154,10 +154,7 @@ Removed
 Porting to Python 3.10
 ======================
 
-This section lists previously described changes and other bugfixes
-that may require changes to your code.
-
-
+* Add :func:`os.cpu_count()` support for VxWorks RTOS.
```

Comment:
Please add this under `Improved Modules`section :)

Reply:
Moved. Thanks.